### ISSUE TITLE
Hover state does not have enough color contrast to differentiate

### ISSUE THREAD

**Issue by meganem** [irinaz/color-buttons-drupal7-admin-interface#2]
Very low contrast between button colors and the hover state color. 
Consider for accessibility adding an extra visual indicator (e.g. underline, or arrow) to the hover/focus state, and/or a much more drastic change in background color (go to dark gray on hover or almost black for example)
![screen shot 2015-10-14 at 1 20 44 pm](https://cloud.githubusercontent.com/assets/1919260/10496619/8c082242-7276-11e5-9bcc-8525d76aaa83.png)
![screen shot 2015-10-14 at 1 20 47 pm](https://cloud.githubusercontent.com/assets/1919260/10496620/8c0924b2-7276-11e5-9c8e-6d2fa8173cba.png)
